mne/forward/forward.py

```diff
@@ -1138,21 +1140,21 @@ def apply_forward(fwd, stc, evoked_template, start=None, stop=None,
     --------
     apply_forward_raw: Compute sensor space data and return a Raw object.
     """
+    if info is None:
+        logger.warning('"evoked" is deprecated and will be removed in '
+                       'MNE-0.11. Please give evoked.info instead')
+        info = evoked_template.info
 
     # make sure evoked_template contains all channels in fwd
     for ch_name in fwd['sol']['row_names']:
-        if ch_name not in evoked_template.ch_names:
+        if ch_name not in info['ch_names']:
             raise ValueError('Channel %s of forward operator not present in '
                              'evoked_template.' % ch_name)
 
     # project the source estimate to the sensor space
     data, times = _apply_forward(fwd, stc, start, stop)
 
-    # store sensor data in an Evoked object using the template
-    evoked = deepcopy(evoked_template)
-
-    evoked.nave = 1
-    evoked.data = data
+    evoked = EvokedArray(data, info, times[0], nave=1)
```

Comment:
You should `deepcopy` the `info` here to make it work the same way. Otherwise modifying the `evoked_output.info` would also modify the `evoked_input.info`.